pkg/storage/unified/sql/queries_test.go

```diff
@@ -52,7 +52,12 @@ func TestUnifiedStorageQueries(t *testing.T) {
 					Data: &sqlResourceRequest{
 						SQLTemplate: mocks.NewTestingSQLTemplate(),
 						WriteEvent: resource.WriteEvent{
-							Key: &resource.ResourceKey{},
+							Key: &resource.ResourceKey{
```

Comment:
Praise: this is great value added to the tests, since now we can see that the generated SQL is putting the values in the correct place.